CHANGELOG.md

```diff
@@ -1,3 +1,27 @@
+# dbt_hubspot v0.25.0
+
+[PR #160](https://github.com/fivetran/dbt_hubspot/pull/160) includes the following updates:
+
+## Breaking Changes
+- Renamed `*_conversion_*` columns in `hubspot__contacts` to avoid conflicts when passthrough columns are enabled.
+
+## Schema Changes
```

Comment:
This is usually listed above Breaking Changes in the new format?

Reply:
Yeah this really isn't necessary since the Schema Changes section already calls this out as well as the breaking changes. Removed and consolidated so this information is included in the schema change section